changelog/unreleased/enhancement-add-accentuated-property

```diff
@@ -0,0 +1,12 @@
+Enhancement: Relative date tooltips
```

Comment:
Minor nitpick: Wouldn't hurt to mention the OcTableFiles component here to make clear where the relative date tooltips are added